pkg/types/builder.go

```diff
@@ -76,12 +74,12 @@ func (g *Builder) buildResource(res *schema.Resource, names ...string) (*types.N
 		switch {
 		case sch.Computed && !sch.Optional:
```

Comment:
There is some discussion [here](https://github.com/crossplane-contrib/terrajet/issues/27).

The reason we're putting only those into status is that the fields that are computed but also optional are the ones that are defaulted by API server. So, users can give an input there but if they don't, server decides on a value and sends it back, which is then late-initialized by us. So we can't really put those fields under status since users are able to enter input there.